apps/dialtone-documentation/docs/about/whats-new/posts/2025-5-6.md

```diff
@@ -0,0 +1,67 @@
+---
+heading: Breaking change in postcss-responsive-variations plugin
+author: Nina Repetto
+posted: '2025-5-6'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+Hey there! We have an important update to share regarding a breaking change in the latest version of the `postcss-responsive-variations` plugin. This change impacts how responsive variations are defined and used.
+
+For **Dialpad** and **Dialpad Meetings** projects, no updates are necessary as we have already updated these projects to use the latest version of the plugin, but we wanted to inform you about these changes so you can use the responsive classes in the new recommended way moving forward.
+
+## What's Changed
+
+The responsive variations on Dialtone classes are the ones that are prefixed with `sm:`, `md:`, `lg:`, and `xl:`. These prefixes are used to apply styles at different screen sizes. The previous implementation used `max-width` media queries, which is not the recommended approach for responsive design.
+The new implementation uses `min-width` media queries, which is more in line with modern responsive design practices. This change allows for a more consistent and predictable behavior when applying styles at different breakpoints.
```

Comment:
Let's also show an example of how breakpoint css classes are used, I think a lot of developers have never used them and won't really know what these are when we're referring to 'md:' 'sm:' etc

Just something simple like this
```
// gap of 16 at screen widths large or greater, gap of 4 otherwise
<div class="d-d-flex d-g4 lg:d-g16">
```